unittests/ADT/IteratorTest.cpp

```diff
@@ -98,4 +98,73 @@ TEST(PointeeIteratorTest, SmartPointer) {
   EXPECT_EQ(End, I);
 }
 
+TEST(FilterIteratorTest, Lambda) {
```

Comment:
Just curious, are these unit tests run as part of the dxc testing infrastructure?